### PR TITLE
X-ray guns have 60% armour penetration.

### PR DESCRIPTION
**Pull request by AnCopper**
### About The Pull Request
gives xray lasers 60 armor penetration

https://github.com/tgstation/tgstation/pull/65629
### Why It's Good For The Game
an xray laser passes through solid objects, armor is a solid object so it makes sense xrays pass through it
xray doesnt really have that much of a niche, it literally does 5 less damage than roundstart laser guns while being late-game research. you kinda do need xray to use it effectively. this should give it a niche outside that and its pretty cool i think

### Changelog
:cl: Fikou, AnCopper
balance: xray lasers now have 60 armor penetration
/:cl:

### CODE REVIEW

**Comment by mcmeiler** [comment]
You copy/pasted the entire PR body from the original tgstation PR.
https://github.com/tgstation/tgstation/pull/65629

**Comment by AnCopper** [comment]
> You copy/pasted the entire PR body from the original tgstation PR. [tgstation/tgstation#65629](https://github.com/tgstation/tgstation/pull/65629)

Yeah I credited Fikou in the changelog, forgot to add the PR to the desc.

**Comment by PowerfulBacon** [comment]
Despite this only really making it more effective against insanely heavilly armoured targets such as deathsquads, I already think its strong enough considering usually the person firing at you is behind a solid wall with thermal vision meaning you literally cannot fight them. Despite having low damage compared to other guns, the user cannot be attacked. Perhaps it can be rebalanced in a different way

**Comment by zeskorion** [comment]
what if it had a scoped mode, like snipers, and that scope gave thermals or xray when in use

**Comment by Oscari-pog** [comment]
No niche
Shoots through walls

What

**Comment by Isy232** [comment]
Thermal vision is an entire different research

It'd be nice to have a conventional weapon thats good at taking down heavily armoured opponents without relying on _other_ means.

**Comment by PowerfulBacon** [comment]
:key:

**Review by Archanial — DISMISSED** [review]

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -62,6 +62,7 @@
 	damage = 15
 	irradiate = 300
 	range = 15
+	armour_penetration = 100
```
Comment (PowerfulBacon) [inline code/modules/projectiles/projectile/beams.dm]: How about buffing the damage, lowering armour penetration and making its armour flag check for radiation suit protection?

**Review by PowerfulBacon — DISMISSED** [review]
Lower the armour penetration value to something more reasonable, and I'll merge. 100% is just too powerful, even if these are hard to come by.

**Comment by PowerfulBacon** [comment]
Go 60 and I'll merge

**Comment by AnCopper** [comment]
> Go 60 and I'll merge

Man, this is like an auction.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**